haskell/test-common/Lorentz/Contracts/Test/Common.hs

```diff
@@ -147,11 +147,14 @@ defaultOriginationParams = OriginationParams
   , opPaused = False
   , opMinters = mempty
   , opPendingOwner = Nothing
-  , opMetadataUri = CurrentContract (metadataJSON $ Just testFA2TokenMetadata) True
+  , opMetadataUri = CurrentContract metadata True
   , opTransferlistContract = Nothing
   , opDefaultExpiry = 1000
   , opPermits = mempty
   }
+  where
+    metadata = either (error . show) id $ metadataJSON (Just testFA2TokenMetadata)
```

Comment:
In nettests you usually use `failure . build`, so why not use `fmt` here as well:
```suggestion
    metadata = either (error . pretty) id $ metadataJSON (Just testFA2TokenMetadata)
```

Reply:
Done